cfy_manager/components/restservice/db.py

```diff
@@ -75,9 +74,10 @@ def _create_args_dict(full_config=False):
     script that creates and populates the DB to run
     """
     args_dict = {
-        'amqp_host': config[RABBITMQ][ENDPOINT_IP],
+        'amqp_host': '127.0.0.1',
```

Comment:
Why? What if we rabbit isn't running on the same VM?

Reply:
because we no longer expose the management api to the outside, only on localhost. What if it's on another vm, should we add a separate config setting for the api?

Reply:
I think so. A random user shouldn't have access to it anyway, because he'll need to authenticate, but we shouldn't be going in the opposite direction of a more distributed infrastructure.